### ISSUE TITLE
GMT5_SHAREDIR set by cpack

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#944]
**Description of the problem**

```
cmake/dist/CMakeLists.txt
155:	set (CPACK_NSIS_ADD_ENVVAR1 "GMT5_SHAREDIR")

cmake/modules/NSIS.InstallOptions.ini.in
6:Text=By default @CPACK_PACKAGE_NAME@ adds its binary directory to the system PATH and sets GMT5_SHAREDIR.
```
It seems the Windows installer will set GMT5_SHAREDIR. Is it correct? Change it to GMT_SHAREDIR?

**Comment by joa-quim** [comment]
Good that you remind this. ``GMT5_SHAREDIR`` was introduced to tell from the GMT4 ``GMT_SHAREDIR`` and is searched first (the `5`). If we make it so (setting ``GMT_SHAREDIR``) than any parallel GMT5 share dir would be found first.

Maybe we should add a ``GMT6_SHAREDIR``. And before you ask, this ENV var is need for example in Mirone and the gmtmex.

**Comment by PaulWessel** [comment]
Yes, if needed, definitely look for GMT6_SHAREDIR first.

**Comment by seisman** [comment]
Closed by #946
